Examples/Swift/ManageOfflineRegionsExample.swift

```diff
@@ -0,0 +1,190 @@
+            tableView.trailingAnchor.constraint(equalTo: mapView.trailingAnchor),
+            tableView.heightAnchor.constraint(equalTo: mapView.heightAnchor, multiplier: 0.4),
+            downloadButton.topAnchor.constraint(equalTo: mapView.topAnchor, constant: 100),
+            downloadButton.leadingAnchor.constraint(equalTo: mapView.leadingAnchor, constant: 5),
+            downloadButton.trailingAnchor.constraint(equalTo: mapView.leadingAnchor, constant: 150)
+        ])
+
+    }
+
+    /*In a normal app, the cache/ pack removal would be done within
+ viewWillDisappear, however, for the purposes of our app, we are calling it
+ within viewWillAppear to clear the cache whenever this particular example is
+ loaded. */
+    override func viewWillAppear(_ animated: Bool) {
+        MGLOfflineStorage.shared.resetDatabase { (error) in
+            if let error = error {
+                print(error)
+            } else {
+                MGLOfflineStorage.shared.reloadPacks()
+            }
+        }
+    }
+
+    // Create the tableview to display the downloaded regions
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+
+        if let packs = MGLOfflineStorage.shared.packs {
+            return packs.count
+        } else {
+            return 1
+        }
+    }
+
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let label = UILabel()
+        if section == 0 {
+            label.backgroundColor = UIColor.white
+            label.textColor = UIColor.black
+            label.textAlignment = .center
+            label.text = "No Offline Packs Saved"
+            return label
+        }
+        return nil
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+
+        let cell = UITableViewCell.init(style: .subtitle, reuseIdentifier: "cell")
+        
+        if let packs = MGLOfflineStorage.shared.packs {
+            let pack = packs[indexPath.row]
+            
+            cell.textLabel?.text = "Region \(indexPath.row + 1): size: \(pack.progress.countOfBytesCompleted)"
+            cell.detailTextLabel?.text = "Percent completion: \(pack.progress.percentCompleted)%"
+
+        }
+
+        return cell
+
+    }
+
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        guard let packs = MGLOfflineStorage.shared.packs else { return }
+        if let selectedRegion = packs[indexPath.row].region as? MGLTilePyramidOfflineRegion {
+            mapView.setVisibleCoordinateBounds(selectedRegion.bounds, animated: true)
+        }
+    }
+
+    @objc func startOfflinePackDownload() {
+
+        // Create a region that includes the current viewport and any tiles needed to view it when zoomed further in.
+        // Because tile count grows exponentially with the maximum zoom level, you should be conservative with your `toZoomLevel` setting.
```

Comment:
```suggestion
        /**
         Create a region that includes the current map camera, to be captured
         in an offline map. Note: Because tile count grows exponentially as zoom level
         increases, you should be conservative with your `toZoomLevel` setting.
         */
```